test/sql.test.js

```diff
@@ -270,6 +293,57 @@ describe('sql connector', function() {
     expect(orderBy).to.eql('ORDER BY `NAME` ASC,`VIP` DESC');
   });
 
+  it('builds order by with non existent field throws', function() {
+    expect(() => {
+      connector.buildOrderBy('customer', ['nam?e', 'name']);
```

Comment:
I have mixed feeling about the value `nam?e`, I believe it's not a valid property name in LoopBack. Can you please use a different property name, for example `unknown`? 

```suggestion
      connector.buildOrderBy('customer', ['unknown', 'name']);
```